src/js/themes/darkTheme/colors.ts

```diff
@@ -39,9 +39,11 @@ export const colors: ThemeColors = {
   },
   primary: {
     default: '#1098FC',
+    defaultHover: '#0092FA',
+    defaultPressed: '#54B6FC',
```

Comment:
Matches figma and has css in js and css variable token

![Screenshot 2024-01-18 at 5 57 00 PM](https://github.com/MetaMask/design-tokens/assets/8112138/1f9acfd3-c850-4517-b9aa-61094add7f50)